components/mediation-admin/org.wso2.carbon.rest.api/src/main/java/org/wso2/carbon/rest/api/APIData.java

```diff
@@ -39,6 +39,10 @@
 
 	private boolean isTracingEnable;
 
+	private String version;
+
+	private String versionType;

```

Comment:
Shall we describe version and version type variables? Specially what versionType is?